### PR TITLE
fix: hover aero style

### LINKED ISSUE #2498: Intorducing Aero CSS
We got a nice navbar (thx @prachi00) with a new update, which looks pleasing to my eyes. 

Screenshot 2022-03-04 at 20 22 59

I want to utilize this so-called "Aero CSS" (Inspired by Windows) in the other parts of our app.

- [ ] Use Aero effect for Card detail
- [ ] Use Aero effect for login (wallet select) modal
- [ ] Use Aero effect for loader (if not implemented yet)

Moodboard: 
Screenshot 2022-03-04 at 20 21 40

Screenshot 2022-03-04 at 20 28 07

https://regen-vs-degen.com

Screenshot 2022-03-04 at 20 55 12

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2498
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

image

image

### CODE REVIEW

**Comment by vikiival** [comment]
Screenshot 2022-03-06 at 10 32 52

This is nice

File: components/common/WalletModal.vue [hunk 1]
```diff
@@ -135,14 +135,17 @@ export default class extends Vue {
 .wallet {
   border-radius: 10px;
   max-width: 400px;
+  &.modal-card {
+    background: rgba(12, 12, 12, 0.7);
```
Comment (vikiival) [inline components/common/WalletModal.vue:139]: Please make it as `.scss` variable

**Review by vikiival — CHANGES_REQUESTED** [review]
two things:
1. this (`rgba(12, 12, 12, 0.7);`) should become as variable
2. Wallet modal does not fit in the style

Check for example loader: it has sharp corners and and pink borders.
On the other hand wallet modal has rounded corners


Please make sure that WalletModal follows the same UI style

File: components/common/WalletModal.vue (hunk 1 above)
Comment (Jarsen136) [reply components/common/WalletModal.vue:139]: ok

**Comment by Jarsen136** [comment]
> two things:
> 
> 1. this (`rgba(12, 12, 12, 0.7);`) should become as variable
> 2. Wallet modal does not fit in the style
> 
> Check for example loader: it has sharp corners and and pink borders. On the other hand wallet modal has rounded corners
> 
> Please make sure that WalletModal follows the same UI style
Thanks!
1. I had changed it.
2. Wallet modal do fit the style and u can look at my screenshot.

**Comment by vikiival** [comment]
> 2. Wallet modal do fit the style and u can look at my screenshot.

Sir, I do not want to be rude but do these squares look same to you?
untitled-3

**Comment by Jarsen136** [comment]
> Referen

Oh, I see. sorry, just give me some time to fix it.

**Comment by vikiival** [comment]
> Oh, I see. sorry, just give me some time to fix it.

No problemo 👍

**Comment by vikiival** [comment]
Why
Screenshot 2022-03-07 at 18 23 29

**Comment by vikiival** [comment]
This look super hot
two last things:
**1. Can we have these buttons with sharp edges and dark background?
Screenshot 2022-03-07 at 18 24 22

Moreover can we have modal with sharp edges? 
Screenshot 2022-03-07 at 18 24 50


That's all, can't wait to merge it 💘

**Review by vikiival — CHANGES_REQUESTED** [review]
last CSS magick

**Comment by Jarsen136** [comment]
> 

how about this?  @vikiival 
image
image

**Comment by vikiival** [comment]
Yup, awesome work mate!

**Review by vikiival — APPROVED** [review]
🎨 code looks LGTM

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 128.66 USD/KSM ~ 0.777 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xfa908aa03010327f298be0cb88f207d8e4537897d6b4d968a228f197ac6d1c70](https://kusama.subscan.io/extrinsic/0xfa908aa03010327f298be0cb88f207d8e4537897d6b4d968a228f197ac6d1c70)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**